storops/unity/resource/lun.py

```diff
@@ -515,6 +529,9 @@ def replicate_with_dst_resource_provisioning(self, max_time_out_of_sync,
             self._cli, self.get_id(), dst_resource, max_time_out_of_sync,
             remote_system=remote_system, name=replication_name)
 
+    def remove_snap_schedule(self):
+        return self.modify(is_snap_schedule_paused=False)
```

Comment:
Should this be True?

Reply:
`False` is correct. I am confusing about this too. But I dumped the REST request sent by Unisphere. `False` is used there too.

I am consulting with the feature team about the detail. Will update this comment once I have the answer.